From 05f9949e837dc04ca2c04d1a085322253f2b1d74 Mon Sep 17 00:00:00 2001
From: debbuild <debbuild>
Date: Sat, 4 Feb 2017 13:12:01 +0100
Subject: [PATCH 03/15] Suppress GCC warning in Dependencies.

---
 test/Singleton/Dependencies.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/Singleton/Dependencies.cpp b/test/Singleton/Dependencies.cpp
index 6bf1e38..203b0e6 100644
--- a/test/Singleton/Dependencies.cpp
+++ b/test/Singleton/Dependencies.cpp
@@ -454,6 +454,8 @@ int main()
     system("PAUSE");
 #endif
 
+    (void)p; // GCC warning suppressed
+
     std::cout << "\nnow leaving main \n";
     std::cout << "________________________________\n\n";
 
-- 
2.20.1

