From f713616a4e3734bb6a145786f27ae9a7b41f085d Mon Sep 17 00:00:00 2001
From: debbuild <debbuild>
Date: Sat, 4 Feb 2017 13:12:20 +0100
Subject: [PATCH 04/14] Hard type conversion in timer.

---
 test/SmallObj/timer.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/SmallObj/timer.h b/test/SmallObj/timer.h
index 1b96440..72e30d0 100644
--- a/test/SmallObj/timer.h
+++ b/test/SmallObj/timer.h
@@ -31,12 +31,12 @@ public:
 
     void start()
     {
-        t0 = clock();
+        t0 = static_cast<int>(clock());
     }
 
     void stop()
     {
-        t1 = clock();
+        t1 = static_cast<int>(clock());
     }
     
     int t()
-- 
2.20.1

