From 67a3a041e39dd0e7fef148c3e59c1c8588b525ce Mon Sep 17 00:00:00 2001
From: debbuild <debbuild>
Date: Sat, 4 Feb 2017 13:12:45 +0100
Subject: [PATCH 05/11] Suppress two GCC warnings in main.

---
 test/SmartPtr/main.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/SmartPtr/main.cpp b/test/SmartPtr/main.cpp
index e5b1c17..6eec6ed 100644
--- a/test/SmartPtr/main.cpp
+++ b/test/SmartPtr/main.cpp
@@ -776,7 +776,7 @@ void DoRefLinkSwapTests( void )
     merged = p9.Merge( p2 );
     // p7 <---> p6 <---> p8 <---> p3 <---> p5 <---> p2 <---> p9 <---> pA
     //   and    p4 <---> p1
-    assert( merged );
+    assert( merged ); (void)merged; // GCC warning suppressed
     assert( p6 == p8 );
     assert( p6 == p7 );
     assert( p1 != p7 );
@@ -1038,6 +1038,8 @@ int main( int argc, const char * argv[] )
         DoLockedStorageTest();
         DoLockedPtrTest();
     }
+#else
+    (void)doThreadTest; // GCC warning suppressed
 #endif
 
     // Check that nothing was leaked.
-- 
2.11.0

